assets/terraform-modules/packet/flatcar-linux/kubernetes/bootkube.tf

```
@@ -42,7 +42,8 @@ module "bootkube" {
   # metadata.packet.net should always resolve to 192.80.8.124.
   blocked_metadata_cidrs = ["192.80.8.124/32"]
 
-  bootstrap_tokens     = var.enable_tls_bootstrap ? concat([local.controller_bootstrap_token], var.worker_bootstrap_tokens) : []
-  enable_tls_bootstrap = var.enable_tls_bootstrap
-  encrypt_pod_traffic  = var.encrypt_pod_traffic
+  bootstrap_tokens            = var.enable_tls_bootstrap ? concat([local.controller_bootstrap_token], var.worker_bootstrap_tokens) : []
+  enable_tls_bootstrap        = var.enable_tls_bootstrap
+  encrypt_pod_traffic         = var.encrypt_pod_traffic
+  failsafe_inbound_host_ports = []
```

Can we have a comment here please, explaining the value?
```suggestion
  # We install calico-host-protection chart on Packet which ships GNPs, so we can disable failsafe ports in Calico.
  failsafe_inbound_host_ports = []
```